po/de.po

```diff
@@ -130,8 +130,9 @@ msgstr "Der Ausdruck %s ist falsch."
 
 #: src/string/i18n.c:15
 #, c-format
-msgid "The expression (as strings) %s is false."
-msgstr "Der Ausdruck (als Zeichenkette) %s ist falsch."
+msgid "The expression (as strings) %s is false: actual=`%s` expected=`%s`."
+msgstr ""
+"Der Ausdruck (als Zeichenkette) %s ist falsch: actual=`%s` expected=`%s`."
```

Comment:
@kaidowei AFAIK is the "actual" string the one you want to check against the expected string.
To pass the test the "actual" string (returned from a function, ...) must be match the expected string ( the one you want to be returned by the function).
One could use "tatsächlich" for the expected string, but I think "erwartet" is the better one.